src/world/__init__.py

```diff
@@ -10,14 +11,26 @@ def calculate_development(self):
         self.calculate_population_development()
 
     def calculate_population_development(self):
-        for population in self.populations:
-            population.calculate_development()
+        self.calculate_available_food()
         for population in self.populations:
```

Comment:
Why do you think its better to first calculate decline, apply it and the calculate incline?